server/service/GithubService.js

```diff
@@ -191,23 +192,37 @@ export class GithubService {
   }
 
   async getComments(user, repo, number, since, accessToken) {
-    let path = API_URL_TEMPLATES.COMMENT
-                                .replace('${owner}', user)
-                                .replace('${repo}', repo)
-                                .replace('${number}', number)
-    if (since) {
-      path += `?since=${since}`
-    }
+    const basePathFromTemplate = API_URL_TEMPLATES.COMMENT
+                                                  .replace('${owner}', user)
+                                                  .replace('${repo}', repo)
+                                                  .replace('${number}', number)
+    const path = `${basePathFromTemplate}?per_page=100`
```

Comment:
By default GitHub return the first 30 entries. 100 is the maximum. Also since query param is not described in the docs and doesn't work (hence filtering was written before, I assume).